lib/onboarding/widgets/select-female-voice.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:flutter/material.dart';
+import 'package:cboard_mobile/stylesheets/typography.dart' as CustomTypography;
+import 'package:flutter/foundation.dart';
+import 'package:cboard_mobile/onboarding/widgets/checkmark.dart';
+
+class SelectFemaleVoice extends StatefulWidget {
+  const SelectFemaleVoice({Key key}) : super(key: key);
+  @override
+  _SelectFemaleVoiceState createState() => _SelectFemaleVoiceState();
+}
+
+class _SelectFemaleVoiceState extends State<SelectFemaleVoice> {
+  bool _selectedVoice = false;
+  void _toggle() {
+    setState(() {
+      _selectedVoice = !_selectedVoice;
+    });
+  }
+
+  Widget build(BuildContext context) {
+    return Padding(
+        padding: EdgeInsets.only(top: 5.0, bottom: 5.0, left: 5.0, right: 5.0),
```

Comment:
<img width="347" alt="Screen Shot 2021-08-16 at 2 18 44 AM" src="https://user-images.githubusercontent.com/59303921/129519811-501cd1dc-6cee-4591-99aa-7e010c6f3c1f.png">

This component appears to be overflowed on my end. I think maybe because of this fixed padding?